vendor/geoip2/geoip2/.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
brings in a lot of dev files which are not meant to ship.
So either the exclude list on packaging needs heavy additions, or we revert all changes on vendor/ and install the dev dependency only on the cs-checker. (I was actually too lazy for that and that is why this app did not use psalm so far

Reply:
Ahaha, I fell into the trap
I thought that was gonna be an easy task
let me remove php-cs for now